src/codeu/chat/common/User.java

```diff
@@ -25,6 +27,11 @@
 
 public final class User {
 
+  public Time updateTime = Time.now();
+  HashMap<Uuid, Integer> conversationInterests
```

Comment:
I hadn't even noticed I did this, thanks for pointing it out! I updated my code.